pre-commit.sh.tpl

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Another idea: we could write this in python (that is, if you'd like to :), since it's a must anyway.

Reply:
Agree with you, did that